GameOfLife/GameOfLife/Board.cs

```diff
@@ -0,0 +1,166 @@
+        /// </summary>
+        private bool[,] initialBoard;
+
+        private readonly int _height;
+        private readonly int _width;
+
+
+        /// <summary>
+        /// Creates two-dimensional array with user-desired sizes
+        /// and populates it with randomly generated boolean values.
+        /// </summary>
+        /// <param name="height"></param>
+        /// <param name="width"></param>
+        public Board(int height, int width)
+        {
+            initialBoard = new bool[height, width];
+
+            _height = height;
+            _width = width;
+
+            Random rnd = new Random();
+
+            for (int i = 0; i < height; i++)
+            {
+                for(int j = 0; j < width; j++)
+                {
+                    initialBoard[i, j] = rnd.Next(0, 2) == 1 ? true : false;
+                }
+            }
+
+        }
+        
+        /// <summary>
+        /// Constructor for testing purposes.
+        /// </summary>
+        /// <param name="layout"></param>
+        public Board(bool[,] layout)
+        {
+            initialBoard = layout;
+
+            _height = layout.GetLength(0);
+            _width = layout.GetLength(1);
+        }
+
+        /// <summary>
+        /// Starts game. Game played infinitely.
+        /// </summary>
+        public void Play()
+        {
+            do
+            {
+                DisplayBoard();
+                Iterate();
+
+                Thread.Sleep(1000);
+                Console.Clear();
+
+            } while (true);
+
+        }
+
+        /// <summary>
+        /// Iterates whole board of cells one time. 
+        /// Results in newer generation of live cells based on rules.
+        /// </summary>
+        public void Iterate()
+        {
+            bool[,] newBoard = new bool[_height, _width];
+
+            for(int i = 0; i < initialBoard.GetLength(0); i++)
+            {
+                for (int j = 0; j < initialBoard.GetLength(1); j++)
+                {
+                    int aliveCellCount = Count(i, j);
+
+                    bool newCell = Determine(aliveCellCount, initialBoard[i, j]);
+                    newBoard[i,j] = newCell;
+                }
+            }
+
+            initialBoard = newBoard;
+        }
+
+        /// <summary>
+        /// Displays board on console.
+        /// Live cells are indicated by white square and un-alived cells by nothing.
+        /// </summary>
+        public void DisplayBoard()
+        {
+            for (int i = 0; i < initialBoard.GetLength(0); i++)
+            {
+                for (var j = 0; j < initialBoard.GetLength(1); j++)
+                {
+                    Console.Write(initialBoard[i, j] ? (char)009632 + " " : "  ");
+                }
+                Console.WriteLine();
+            }
+        }
+
+        /// <summary>
+        /// Iterates through all of the neighbours of a given cell indices.
+        /// </summary>
+        /// <param name="x"></param>
```

Comment:
Missing param comment.

Reply:
And what is x ?

Reply:
x indicated the row and y indicated the column in which cell could be found but as it was not very understandable I changed them x to row and y to column. Param comment added.